### DISCUSSION TITLE
Using CSS functions in bound attributes

### DISCUSSION THREAD

**Discussion by coder2000** [alpinejs/alpine#3857] [1. Help] [answered]
Is it possible to use a CSS function in a bound attribute like: `<div :style="{'background-color': rgba(variable, 1.0)}">Content</div>`

**Comment by imacrayon** [comment, marked as answer]
Yes, the CSS properties and values just both needs to be a string:

```html
<div x-data="{ variable: '255, 0, 0' }" :style="{'background-color': `rgba(${variable}, 1.0)`}">Content</div>
```

**Reply by coder2000** [reply]
I had tried that and it didn't seem to work but it may have been that my variable was already wrapped in the rgb function. Thanks for the help.

**Reply by ekwoka** [reply]
Additionally, you can use the actual style property name `backgroundColor` without needing to wrap it as a string.

You could also have the :style be a singular string 

```html
<div x-data="{ variable: '255, 0, 0' }" :style="`background-color: rgba(${variable}, 1.0);`">Content</div>
```
